WORKSPACE

```diff
@@ -466,3 +466,12 @@ maven_jar(
   artifact = "com.yahoo.pulsar:pulsar-client:jar:shaded:1.18"
 )
 # end Pulsar Client
+
+# for pex rules
+git_repository(
+    name = "io_bazel_rules_pex",
+    remote = "https://github.com/streamlio/bazel_rules_pex.git",
```

Comment:
We need to test if we can build inside of Twitter environment. Inside Twitter, there is no access to outside network. This means we cannot fetch pex rules repo.